src/containers/ConfirmationContainer/components/SuccessModal/SuccessModal.vue

```diff
@@ -42,15 +43,35 @@ export default {
     linkTo: {
       type: String,
       default: '/'
+    },
+    etherscanLink: {
+      type: String,
+      default: null
     }
   },
+  data() {
+    return {
+      buttonCheckEtherscan: {
+        title: 'Check Status on Etherscan',
```

Comment:
It is not always etherscan, we cant have it say `Check Status on Etherscan`

Reply:
may be simply say `Check Status` also, can you change all `etherscan` property names to `explorer` for example `etherscanLink` will become `explorerLink`

Reply:
incoming push should fix this to replace the name depending on network